### PR TITLE
update version to v3.0.0

### PR DESCRIPTION
**Pull request by ohine**
## Summary
Update version for the v9.4.0 Platform release.

### CODE REVIEW

**Review by mtrutledge — APPROVED** [review]
Looks good

**Comment by bdukes** [comment]
This is going to conflict with #711.  Would it be easier to wait until that gets merged into [`development`](https://github.com/dnnsoftware/Dnn.AdminExperience/tree/development)?

**Review by bdukes — APPROVED** [review]

**Comment by ohine** [comment]
@bdukes I wasn't sure if it would conflict since it is from the branch and already includes the commits to v2.0.0 and v2.0.1. We can wait if you think that's safer.

**Pull request merged**
